imjoy/engine.py

```diff
@@ -18,25 +18,35 @@ def __init__(self, opt, logger):
         self.conn = None
         self.store = None
 
+    def __repr__(self):
+        """Return the engine representation."""
+        return f"<Engine(opt={self.opt})>"
+
+    @classmethod
+    def create(cls, args):
+        """Prepare and and return an Engine instance."""
+        logging.basicConfig(stream=sys.stdout)
+        logger = logging.getLogger("ImJoyPluginEngine")
+        opt = parse_cmd_line(args)
+        setup_logging(opt, logger)
+        opt = prep_env(opt, logger)
```

Comment:
how about 'prepare_env'?

Reply:
Yeah, but I don't want to change more things outside the scope of this PR.